tests/comdb2sys.test/comdb2sys.expected

```diff
@@ -417,3 +417,10 @@
 (name='comdb2_users')
 (name='comdb2_views')
 [SELECT * FROM comdb2_systables ORDER BY name] rc 0
+[DROP TABLE IF EXISTS tbase1] rc 0
+[CREATE TABLE tbase1 (i INT PRIMARY KEY)] rc 0
+[CREATE TIME PARTITION ON tbase1 AS tview1 PERIOD 'YEARLY' RETENTION 1 START '2038-01-01T00:00:00 UTC'] rc 0
```

Comment:
Do you mind using the new syntax here?
https://bloomberg.github.io/comdb2/table_partition.html